src/components/Frame/components/Toast/Toast.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import {classNames} from '@shopify/react-utilities';
 
 import {Key} from '../../../../types';
+import Link from '../../../Link';
```

Comment:
this should probably be a `Button`, no?

Reply:
Could you develop more on that, please?

Reply:
generally a Link is for something that takes you somewhere whereas a Button is for something that performs an action

Reply:
Cool, I'll make the change, thank you for the insight 👍